neptun/src/noise/timers.rs

```diff
@@ -82,6 +96,7 @@ pub struct Timers {
     persistent_keepalive: usize,
     /// Should this timer call reset rr function (if not a shared rr instance)
     pub(super) should_reset_rr: bool,
+    timers_to_update_mask: AtomicU16,
```

Comment:
do we need atomicu16 here? can we use instead the `parking_lot::Mutex<u16>`  or even `parking_lot::Mutex<[bool;8]>`?

Reply:
So this is main area that should be bringing improment, as then we are just swtiching from a mutex to a mutex.
Now we fundamentaly move away from any wait for lock's.

Reply:
This change is adding atomicu16, not removing any mutex

Reply:
Oh right, it's not with self as &self, but &mut self.
We don't even need Mutex or Atomic then in reality, just u16 would be enough